pex/vendor/README.md

```diff
@@ -0,0 +1,37 @@
+PEX Vendored Distributions
+==========================
+
+PEX vendors distributions of critical third party code it uses at build-time and at run-time both
+to ensure predictable behavior and provide a zero-dependency library and cli tool to build upon
+without fear of dependency conflict by higher software layers.
+
```

Comment:
I'm not going to go there on patches because I agree very strongly with just never editing period. Added a comment that this is the point - vendor as is, rewrite imports mechanically - you want a fix contribute it upstream and re-vendor.